precompiles/outposts/osmosis/osmosis.go

```diff
@@ -47,6 +47,7 @@ var f embed.FS
 // the common Precompile type.
 type Precompile struct {
 	cmn.Precompile
+	WevmosAddress common.Address
```

Comment:
can this be a private field instead of public?
With the current logic, will work the same as private. Unless we're planning future use cases that will require it to be public